src/iop/atrous.c

```diff
@@ -1766,6 +1766,23 @@ const dt_action_def_t _action_def_equalizer
       _action_elements_equalizer,
       _action_fallbacks_equalizer };
 
+
+static void _ui_pipe_done(gpointer instance, dt_iop_module_t *self)
+{
+  dt_iop_atrous_gui_data_t *g = self->gui_data;
+  if(!g || darktable.gui->reset) return;
+
+  if(dt_dev_gui_module() == self)
```

Comment:
Maybe we want to refresh if the module is expanded and enabled? Because if you open a another IOP and still see the contrast equalizer UI it seems better to also refresh it?